### DISCUSSION TITLE
Link the GitHub CI checks directly to the Encore build/deployment

### DISCUSSION THREAD

**Discussion by macleodmac** [encoredev/encore#1066] [Suggestions]
Currently, on CI checks we need to click twice to get to the relevant build/deploy.
1.
![image](https://github.com/encoredev/encore/assets/8996312/4be8ef44-6007-49cf-ac5c-a06a722c0826)

2.
![image](https://github.com/encoredev/encore/assets/8996312/aab85c23-add1-472a-b542-da50d490054c)

It would be great to set the `details_url` on the [GitHub check run](https://docs.github.com/en/rest/checks/runs?apiVersion=2022-11-28#create-a-check-run) so that we can click through directly to the relevant build/deploy.

**Comment by eandre** [comment]
We already set the details url as you describe, as far as I can tell. Do you have an example of a GitHub that has the behavior you describe? As far as I know there isn't the ability to do what you're asking, and setting the `details_url` field is what enables the "view more details" link to work in the first place.

**Reply by macleodmac** [reply]
@eandre you're right - apologies! I was going by the Chromatic app which has this functionality, but I think they're using [Commit Statuses](https://docs.github.com/en/rest/commits/statuses?apiVersion=2022-11-28#create-a-commit-status) instead of Check Runs.
